### ISSUE TITLE
GraphiQL 2.0

### ISSUE THREAD

**Issue by agriffard** [OrchardCMS/OrchardCore#12295]
If anyone wants to make the upgrade for GraphiQL to version 2.0.x, please comment.

Have the look at the breaking changes:
https://github.com/graphql/graphiql/releases/tag/graphiql%402.0.0

**Comment by Skrypt** [comment]
Yes, we can't make everything backward compatible. This is a natural evolution.

**Comment by hishamco** [comment]
Maybe they are related, I might need to check with the help of @hyzx86

**Comment by Piedone** [comment]
Where was this done @agriffard?
